packages/postcss-config/package.json

```
@@ -0,0 +1,38 @@
+{
+	"name": "@wordpress/postcss-config",
+	"version": "0.0.1",
```

Do we consider it experimental?

```suggestion
	"version": "1.0.0",
```

---

We can’t use 1.0.0 because Lerna always bumps version but I can update to 1.0.0-pre or something like that.

---

> We can’t use 1.0.0 because Lerna always bumps version but I can update to 1.0.0-pre or something like that.

Ah right, forgot about that. I guess if it's bumped to 1.0.0, it shouldn't matter either way. In general it might help avoid some attention if we'd standardize on something like "-pre" for review stage.

---

`1.0.0-pre` it is 👍 (c256fc006845815231289f3d42afe12855ccbc4d)